content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
 
-**The `TIMER` metric type is an implementation of the `HISTOGRAM` metric type within DogStatsD** (not to be confused with timers in the standard StatsD). It measures timing data only: for example, the amount of time a section of code takes to execute, or how long it takes to fully render a page. See the [TIMER DogStatsD documentation][1] to learn how to instrument your code to submit `TIMER`s.
+**Note**: When submitting a RATE metric type through DogStatsD, the metric appears as a GAUGE in-app to ensure relevant comparison across different Agents.
 
-[1]: /developers/metrics/dogstatsd_metrics_submission/#timers
+
+[1]: /api/?lang=python#post-timeseries-points
+[2]: /developers/metrics/agent_metrics_submission/?tab=rate
 {{% /tab %}}
-{{< /tabs >}}
+{{% tab "GAUGE" %}}
 
-## Submission types and Datadog in-app types
+Submit your GAUGE type metrics from one of the following sources:
 
-Datadog accepts metrics submitted from a variety of sources:
+| Submission Source | Submission Method (python)           | Submission Type | Datadog In-App Type |
+| ----------------- | ------------------------------------ | --------------- | ------------------- |
+| [API][1]          | `api.Metric.send(type="gauge", ...)` | GAUGE           | GAUGE               |
+| [DogStatsD][2]    | `dog.gauge(...)`                     | GAUGE           | GAUGE               |
+| [Agent check][3]  | `self.gauge(...)`                    | GAUGE           | GAUGE               |
 
-* [Datadog API][3]
-* [DogStatsD][4]
-* [Agent Check][5]
 
-Each source has its own limitations, and metric submission types do not always map exactly to the Datadog in-app stored types:
+[1]: /api/?lang=python#post-timeseries-points
+[2]: /developers/metrics/dogstatsd_metrics_submission/#gauge
+[3]: /developers/metrics/agent_metrics_submission/?tab=gauge
+{{% /tab %}}
+{{% tab "HISTOGRAM" %}}
+
+Submit your HISTOGRAM type metrics from one of the following sources:
+
+| Submission Source | Submission Method (python) | Submission Type | Datadog In-App Type |
+| ----------------- | -------------------------- | --------------- | ------------------- |
+| [DogStatsD][1]    | `dog.histogram(...)`       | HISTOGRAM       | GAUGE, RATE         |
+| [Agent check][2]  | `self.histogram(...)`      | HISTOGRAM       | GAUGE, RATE         |
+
+**Note**: If you submit a TIMER metric to the Datadog Agent, it is equivalent to submitting a HISTOGRAM metric type within DogStatsD (not to be confused with timers in the standard StatsD). Timers represent duration data only; for example, the amount of time a section of code takes to execute or how long it takes to fully render a page.
+
+
+[1]: /developers/metrics/dogstatsd_metrics_submission/#histogram
+[2]: /developers/metrics/agent_metrics_submission/?tab=histogram
+{{% /tab %}}
+{{% tab "DISTRIBUTION" %}}
+
+Submit your DISTRIBUTION type metrics from the following source:
+
+| Submission Source | Submission Method (python) | Submission Type | Datadog In-App Type |
```

Comment:
| Submission Source | Submission Method (python) | Submission Type | Datadog In-App Types |